lib/open-amp/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2020, STMicroelectronics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+zephyr_include_directories_ifdef(CONFIG_OPENAMP_RSC_TABLE .)
```

Comment:
I think we can remove this line now.

Reply:
need to keep it to be able to include resource_table.h in sample